.travis.yml

```diff
@@ -41,3 +46,7 @@ deploy:
 
 addons:
   sonarqube: true
+
+after_success:
+  - docker login -u aschen -p $DOCKER_PASSWORD
```

Comment:
I'd prefer a generic account dedicated to the whole team

Reply:
So I have to create an account. Which email should I use ?